python/ray/data/context.py

```diff
@@ -165,9 +165,6 @@ class ShuffleStrategy(str, enum.Enum):
 # Use this to prefix important success messages for the user.
 OK_PREFIX = "✔️ "
 
-# Default batch size for batch transformations.
-DEFAULT_BATCH_SIZE = 1024
```

Comment:
can we keep this variable so users can still revert to 1024 if they need?

Reply:
Addressed